vault/dendron.topic.images.md

```diff
@@ -2,35 +2,42 @@
 id: a91fd8da-6895-49fe-8164-a17acd8d9a17
 title: Images
 desc: ''
-updated: 1659026813451
+updated: 1662866892933
 created: 1595562035825
-stub: false
 ---
-    
-## Copy & Paste
+## Summary
 
-Dendron automatically inserts images in your clipboard using the `> Paste Image` command. The image is saved to `${currentFileDir}/assets` by default. This feature requires the [Paste Image](https://link.dendron.so/vscode-paste-image) VS Code extension.
+Dendron extends regular Markdown image support by allowing custom CSS properties. Images can also be pasted into a file from your clipboard.
+
+## Use Cases
+
+* add images and screenshots to notes
+* modify images using inline YAML map
+
+## Features
+### Copy & Paste
+
+Dendron automatically inserts images in your clipboard using the `> Paste Image` command. The image is saved to `${currentFileDir}/assets/images` by default. This feature requires the [Paste Image](https://link.dendron.so/vscode-paste-image) VS Code extension.
 
 <a href="https://www.loom.com/share/e1f6d207a1134f42b7a1a7750658acec">
 <img style="" src="https://cdn.loom.com/sessions/thumbnails/e1f6d207a1134f42b7a1a7750658acec-with-play.gif"> </a>
 </a>
 
-## Inserting Saved Images
+### Inserting Saved Images
```

Comment:
this isn't really a feature, more just describing how to link to existing images. rename to "Link to existing images"

Reply:
Hey Kevin, thanks for the feedback. Do you think it might be a good idea to go with something like "locally saved image" rather than "existing image"? This would clarify that the image should be in the vault.

Reply:
sounds good